cernopendata/opendata.cern.ch
# frontend: upgrade search UI

- upgrade to latest invenio-search-ui (2.0.0)
- use default search template
- create react components for various result types
- get rid of unused templates, JS, styles...
- MVP funcionality

closes #3014
addresses https://github.com/cernopendata/opendata.cern.ch/issues/2997

<img width="1165" alt="image" src="https://user-images.githubusercontent.com/8863238/105375159-c5070f00-5c08-11eb-8b23-75b17d50c398.png">

### To test

- Search for a term in the search bar (e.g. http://0.0.0.0:5000/search?q=higgs)
- Check different facets
- Use pagination
- ... try to break it and report issues :)
